pkg/loki/modules.go

```diff
@@ -53,6 +52,7 @@ import (
 	"github.com/grafana/loki/pkg/storage/stores/shipper"
 	"github.com/grafana/loki/pkg/storage/stores/shipper/compactor"
 	"github.com/grafana/loki/pkg/storage/stores/shipper/compactor/deletion"
+	"github.com/grafana/loki/pkg/storage/stores/shipper/compactor/generationnumber"
```

Comment:
nit: indentation

Reply:
This looks the same here and in the editor?

Reply:
yep, just a display issue on my end